rewrite-test/build.gradle.kts

```diff
@@ -12,4 +12,6 @@ dependencies {
     implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-csv")
     implementation("org.slf4j:slf4j-api:1.7.36")
     implementation("org.eclipse.jgit:org.eclipse.jgit:5.13.+")
+
+    testImplementation(project(":rewrite-groovy"))
```

Comment:
Added as a test dependency; for `java(...)` we would have similarly had to add for instance `rewrite-java-17`. Figured have the test below reflect what triggered the change, but open to change this. Didn't spot a second source file type in rewrite-core to use here.

Reply:
This is fine.